### PR TITLE
feat: (TNLT-5066) Front Lead 1 and 1 with ITE

### PR DESCRIPTION
**Pull request by aronshamash**
-  Rebuilds the Front Lead One and One Slice to include In Todays Edition component.
- Uses the new `5:4` image crop for the first time which is currently being sent by Methode as `1.25:1` but is due to be replicated in TPA as `5:4` - https://nidigitalsolutions.jira.com/browse/TNLT-5558
- Some small tweaks to `InTodaysEdition` styles
- https://nidigitalsolutions.jira.com/browse/TNLT-5066

### iPad Mini (Portrait)
Screenshot 2020-10-09 at 17 01 37

### iPad (Portrait)
Screenshot 2020-10-09 at 17 01 56

### iPad Air (Portrait)
 
Screenshot 2020-10-09 at 17 03 43

### iPad Pro  12.9 (Portrait)
Screenshot 2020-10-09 at 17 11 11

### iPad Mini (Landscape)
Screenshot 2020-10-09 at 17 01 51

### iPad (Landscape)
Screenshot 2020-10-09 at 17 17 16

### iPad Air (Landscape)
Screenshot 2020-10-09 at 17 18 43

### iPad Pro 12.9" (Landscape)
Screenshot 2020-10-09 at 17 24 10

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -464,7 +464,7 @@ SPEC CHECKSUMS:
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   CocoaAsyncSocket: 694058e7c0ed05a9e217d1b3c7ded962f4180845
   CocoaLibEvent: 2fab71b8bd46dd33ddb959f7928ec5909f838e3f
-  DoubleConversion: cde416483dac037923206447da6e1454df403714
+  DoubleConversion: 5805e889d232975c086db112ece9ed034df7a0b2
```
Comment (c-j-j) [inline ios/Podfile.lock:467]: Did you mean to commit this file?

**Review by c-j-j — APPROVED** [review]

File: ios/Podfile.lock (hunk 1 above)
Comment (aronshamash) [reply ios/Podfile.lock:467]: no. but I was wondering why only I am getting this update on `pod install`... I will revert that change.

File: packages/front-page/front-article-summary-content/index.tsx [hunk 2]
```diff
@@ -37,7 +34,6 @@ const SummaryText: React.FC<SummaryTextProps> = ({
     </Text>
   ) : null;
 };
-
```
Comment (Geoff-Ford) [inline packages/front-page/front-article-summary-content/index.tsx:40]: Weird that all these lines have been removed. Is this a prettier thing?

File: packages/in-todays-edition/styles/index.js [hunk 3]
```diff
@@ -152,7 +159,7 @@ const styles = {
       },
       itemsContainer: {
         ...sharedPortraitStyles.itemsContainer,
-        marginTop: spacing(2),
+        marginTop: 17,
```
Comment (Geoff-Ford) [inline packages/in-todays-edition/styles/index.js:162]: `spacing(3) + 2`? Not that bothered though.

File: packages/slice-layout/src/templates/frontleadoneandone/index.js [hunk 4]
```diff
@@ -1,42 +1,56 @@
...
     >
-      <View style={styles.leftColumn}>{lead}</View>
-      <ItemColSeparator style={styles.colSeparatorStyle} />
-      <View style={styles.rightColumn}>{support}</View>
-    </View>
+      <View style={styles.tilesContainer}>
+        <View style={styles.leftColumn}>{lead}</View>
+        <ItemColSeparator style={styles.colSeparatorStyle} />
+        <View style={styles.rightColumn}>{support}</View>
+      </View>
```
Comment (Geoff-Ford) [inline packages/slice-layout/src/templates/frontleadoneandone/index.js:51]: Can't the above also be a `HorizontalLayout`?

**Review by Geoff-Ford — APPROVED** [review]

File: packages/in-todays-edition/styles/index.js (hunk 3 above)
Comment (aronshamash) [reply packages/in-todays-edition/styles/index.js:162]: going to leave I think

File: packages/slice-layout/src/templates/frontleadoneandone/index.js (hunk 4 above)
Comment (aronshamash) [reply packages/slice-layout/src/templates/frontleadoneandone/index.js]: yes I've updated

**Pull request merged**